molgenis-app/src/main/resources/logback.xml

```diff
@@ -9,7 +9,4 @@
 	<root level="WARN">
 		<appender-ref ref="STDOUT" />
 	</root>
-	<logger name="org.molgenis.data.jobs.model.JobExecution" level="ALL">
```

Comment:
this logger is used to fill the "log" attribute in the "Annotation job execution (sys_AnnotationJobExecution)". Removing it will result in an empty column after a job execution.